src/ripple/app/book/impl/BookTip.cpp

```diff
@@ -28,6 +28,7 @@ BookTip::BookTip (LedgerView& view, BookRef book)
     , m_valid (false)
     , m_book (getBookBase (book))
     , m_end (getQualityNext (m_book))
+    , m_quality ()
```

Comment:
No, but it was included in 0.27.2 and I wanted to make the inevitable reverse-merge easier. It will go away in a subsequent pull.